src/views/InteractiveMap/CardDialogDataDriven.vue

```diff
@@ -0,0 +1,197 @@
+<template>
+  <div>
+    <v-layout>
+      <v-autocomplete
+        class="mr-2"
+        label="Experiment"
+        :items="experiments"
+        v-model="card.experiment"
+        item-text="name"
+        placeholder="Choose the experiment you wish to simulate..."
+        return-object
+      ></v-autocomplete>
+
+      <v-autocomplete
+        class="ml-2"
+        label="Conditions"
+        :items="conditions"
+        v-model="card.condition"
+        :loading="isLoadingConditions"
+        :disabled="conditions === []"
+        item-text="name"
+        placeholder="Choose the conditions you wish to simulate"
+        return-object
+      ></v-autocomplete>
+    </v-layout>
+
+    <v-progress-linear
+      :indeterminate="true"
+      v-if="isLoadingConditionData"
+    ></v-progress-linear>
+
+    <div v-for="warning in card.conditionWarnings" :key="warning" class="mb-2">
+      <v-alert :value="true" type="warning">
+        {{ warning }}
+      </v-alert>
+    </div>
+
+    <div v-for="error in card.conditionErrors" :key="error" class="mb-2">
+      <v-alert :value="true" type="error">
+        {{ error }}
+      </v-alert>
+    </div>
+
+    <v-layout v-if="card.conditionData">
+      <v-flex class="mr-2">
+        <v-card>
+          <v-subheader>Genotype changes</v-subheader>
+          <v-list
+            v-for="(genotype, index) in card.conditionData.genotype"
+            :key="index"
+          >
+            <v-list-tile v-for="gene in genotype.split(',')" :key="gene">
+              <v-list-tile-content>
+                <v-list-tile-title v-text="gene"></v-list-tile-title>
+              </v-list-tile-content>
+            </v-list-tile>
+          </v-list>
+        </v-card>
+      </v-flex>
+      <v-flex>
+        <v-card class="mx-2">
+          <v-subheader>Measurements</v-subheader>
+          <v-list>
+            <!-- TODO: Handle non-bigg namespaces -->
+            <v-list-tile
+              v-for="measurement in card.conditionData.measurements"
+              :key="measurement.id"
+              :href="`http://bigg.ucsd.edu/search?query=${measurement.id}`"
+              target="_blank"
+            >
+              <v-list-tile-content>
+                <v-list-tile-title>
+                  {{ measurement.id }}: {{ measurement.measurements }} mmol
+                  g<sup>-1</sup> h<sup>-1</sup>
+                </v-list-tile-title>
+              </v-list-tile-content>
+            </v-list-tile>
+          </v-list>
+        </v-card>
+      </v-flex>
+      <v-flex>
+        <v-card class="ml-2">
+          <v-subheader>Medium</v-subheader>
+          <v-list>
+            <!-- TODO: Handle non-chebi namespaces -->
```

Comment:
I think opening an issue rather than just leaving a comment in the source code will help us track this better.